addons/placement/functions/fnc_setupPreview.sqf

```diff
@@ -0,0 +1,53 @@
+#include "script_component.hpp"
+/*
+ * Author: Brett, mharis001
+ * Sets up the placement preview for the given object type.
+ * Will delete current preview object if "" is provided.
+ *
+ * Arguments:
+ * 0: Object Type <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["B_Soldier_F"] call zen_placement_fnc_setupPreview
+ *
+ * Public: No
+ */
+
+params ["_objectType"];
```

Comment:
This could take by default an empty string, therefore, cleaning up code some more.